### ISSUE TITLE
Failed to build package gdal

### ISSUE THREAD

**Issue by coonce** [mxe/mxe#2610]
It looks like gdal is looking for openjpeg 2.0, 2.1, 2.2, or 2.3, but mxe is downloading 2.4: 
https://github.com/mxe/mxe/blob/f879e4ea2303e0db5c26ab3a8d3267ed350c4e0f/src/openjpeg.mk#L8
```
Failed to build package gdal for target x86_64-w64-mingw32.static!
------------------------------------------------------------
 libs="-L/home/daniel/dvlp/mxe/usr/x86_64-w64-mingw32.static/lib -lnetcdf -lmfhdf -ldf -lhdf5_hl -lhdf5 -lpthread -lz -lm -lcurl", includedir="/home/daniel/dvlp/mxe/usr/x86_64-w64-mingw32.static/include"
checking for nc_open in -lnetcdf... no
configure: JasPer (JPEG2000) support disabled.
configure: error: openjpeg.h not found in /home/daniel/dvlp/mxe/usr/x86_64-w64-mingw32.static/include/openjpeg-2.0 or /home/daniel/dvlp/mxe/usr/x86_64-w64-mingw32.static/include/openjpeg-2.1 or /home/daniel/dvlp/mxe/usr/x86_64-w64-mingw32.static/include/openjpeg-2.2 or /home/daniel/dvlp/mxe/usr/x86_64-w64-mingw32.static/include/openjpeg-2.3
make[1]: *** [Makefile:870: build-only-gdal_x86_64-w64-mingw32.static] Error 1
make[1]: Leaving directory '/home/daniel/dvlp/mxe'
real    0m13.282s
user    0m11.415s
sys     0m1.520s
------------------------------------------------------------
[log]      /home/daniel/dvlp/mxe/log/gdal_x86_64-w64-mingw32.static
```

**Comment by 3togo** [comment]
@coonce, 

How to fix it ?

**Comment by jonaski** [comment]
gdal probably needs to be upgraded, it's at version 2.2.4, and the newest is 3.2.2.
did you try do upgrade it?

**Comment by tonytheodore** [comment]
gdal is hard to update, [workaround from last package build](https://github.com/mxe/mxe/releases/tag/build-2021-01-10) was to disable openjpeg support with:
```
make gdal_CONFIGURE_OPTS=--with-openjpeg=no ...
```

**Comment by HizzyCO** [comment]
Even with the work around mine still fails

`[build]       gdal                   i686-w64-mingw32.static

Failed to build package gdal for target i686-w64-mingw32.static!
------------------------------------------------------------
libtool: compile:  i686-w64-mingw32.static-g++ -std=gnu++11 -I/usr/src/mxe/tmp-gdal-i686-w64-mingw32.static/gdal-2.2.4/port -I/usr/src/mxe/tmp-gdal-i686-w64-mingw32.static/gdal-2.2.4/gcore -I/usr/src/mxe/tmp-gdal-i686-w64-mingw32.static/gdal-2.2.4/alg -I/usr/src/mxe/tmp-gdal-i686-w64-mingw32.static/gdal-2.2.4/ogr -I/usr/src/mxe/tmp-gdal-i686-w64-mingw32.static/gdal-2.2.4/ogr/ogrsf_frmts -I/usr/src/mxe/tmp-gdal-i686-w64-mingw32.static/gdal-2.2.4/gnm -I/usr/src/mxe/tmp-gdal-i686-w64-mingw32.static/gdal-2.2.4/apps -DHAVE_MITAB -DHAVE_AVX_AT_COMPILE_TIME -DHAVE_SSSE3_AT_COMPILE_TIME -DHAVE_SSE_AT_COMPILE_TIME -D_WIN32_WINNT=0x0600 -Wall -Wextra -Winit-self -Wunused-parameter -Wformat -Werror=format-security -Wno-format-nonliteral -Wlogical-op -Wshadow -Werror=vla -Wdate-time -Wfloat-conversion -Wmissing-declarations -Wnon-virtual-dtor -Woverloaded-virtual -fno-operator-names -Wsuggest-override -Iogrsf_frmts -Iogrsf_frmts/mem -Iogrsf_frmts/geojson -I/usr/src/mxe/usr/i686-w64-mingw32.static/include/json-c -I. -DHAVE_EXPAT -I/usr/src/mxe/usr/i686-w64-mingw32.static/include -DHAVE_GEOS=1 -I/usr/src/mxe/usr/i686-w64-mingw32.static/include -DGNM_ENABLED -DDONT_DEPRECATE_SPRINTF -I/usr/src/mxe/tmp-gdal-i686-w64-mingw32.static/gdal-2.2.4/port -I/usr/src/mxe/usr/i686-w64-mingw32.static/include -I/usr/src/mxe/usr/i686-w64-mingw32.static -I/usr/src/mxe/usr/i686-w64-mingw32.static/include -I/usr/src/mxe/usr/i686-w64-mingw32.static -I/usr/src/mxe/usr/i686-w64-mingw32.static/include -I/usr/src/mxe/usr/i686-w64-mingw32.static/include -I/usr/src/mxe/usr/i686-w64-mingw32.static/include -I/usr/src/mxe/usr/i686-w64-mingw32.static -I/usr/src/mxe/usr/i686-w64-mingw32.static/include -I/usr/src/mxe/usr/i686-w64-mingw32.static -I/usr/src/mxe/usr/i686-w64-mingw32.static/include -I/usr/src/mxe/usr/i686-w64-mingw32.static -I/usr/src/mxe/usr/i686-w64-mingw32.static/include -DGDAL_COMPILATION -c ogr_xerces.cpp -o ogr_xerces.o
make[3]: Leaving directory '/usr/src/mxe/tmp-gdal-i686-w64-mingw32.static/gdal-2.2.4/ogr'
make[2]: Leaving directory '/usr/src/mxe/tmp-gdal-i686-w64-mingw32.static/gdal-2.2.4'
Makefile:851: recipe for target 'build-only-gdal_i686-w64-mingw32.static' failed
make[1]: *** [build-only-gdal_i686-w64-mingw32.static] Error 2
make[1]: Leaving directory '/usr/src/mxe'
real    1m4.087s
user    2m52.212s
sys     0m15.335s`

**Comment by tonytheodore** [comment]
Try:
```
make gdal_CONFIGURE_OPTS='--with-openjpeg=no --with-cryptopp=no' ...
```

**Comment by HizzyCO** [comment]
what does turning that off do?

**Comment by mabrand** [comment]
Should be fixed by eb7df8f39184da38c47c69e9345f7bf091e165c1 and c2cdd3a06bf9b0e596974d70b241c06889ba16f8.
